platform/linux-generic/odp_ishm.c

```diff
@@ -2142,7 +2162,7 @@ void _odp_ishm_print(int block_index)
 	block = &ishm_tbl->block[block_index];
 
 	_ODP_PRINT("\nSHM block info\n--------------\n");
-	_ODP_PRINT(" name:       %s\n",   block->name);
+	_ODP_PRINT(" name:       %s\n",   block->param_name);
```

Comment:
Maybe the internal unique name could still be printed too since this function appears to be meant for debugging?